src/gdlwidget.hpp

```diff
@@ -39,9 +39,6 @@
 #include <wx/popupwin.h>
 #include <wx/notebook.h>
 #include <wx/dcbuffer.h>
-#if __WXMSW__
-#include <wx/msw/wx.rc>  //should provide manifest, but is linker OK?
```

Comment:
Does it require manifest? AFAIK you can use `windres`...

Reply:
@pjb7687 the widgets have no Look & Feel on Windows, because somewhere in the build process, one shoudl provide a gdl.exe.manifest for the windows executable.
But I cannot find an example of how to do that, wxWidget documentation seems to believe everyone adds manifests just like they are breathing and do not need any explanation or example.
At least, #include <wx/msw/wx.rc> only was a failure.
Probably you can help?

Reply:
Added 2 commits below... It should work now.

Reply:
thanks a lot, the look and feel has improved!